dvc/api.py

```diff
@@ -92,6 +94,175 @@ def read(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
         return fd.read()
 
 
+def get_params(
+    *targets: str,
+    repo: Optional[str] = None,
+    revs: Optional[Union[str, Iterable[str]]] = None,
+    deps: bool = False,
+    stages: Optional[Union[str, Iterable[str]]] = None,
+) -> Dict:
```

Comment:
Hm, didn't we agree to go with porcelain-like instead? Or am I missing something?

Reply:
Not sure I understood what the API would be in porcelain-like then. What would be different?

Reply:
`params_show`?